src/Channels.cpp

```diff
@@ -191,19 +191,41 @@ PVR_RECORDING_CHANNEL_TYPE Channels::GetChannelType(unsigned int uid)
 
 PVR_ERROR Channels::GetChannelGroups(bool radio, kodi::addon::PVRChannelGroupsResultSet& results)
 {
-  // radioGroups.size() will be zero before 5.2 API
-  if (radio && m_radioGroups.size() == 0)
+  const std::unordered_set selectedGroup = radio ? m_radioGroups : m_tvGroups;
```

Comment:
`selectedGroup` might not be a good name. Maybe `selectedGroups`?

Reply:
Makes sense I was thinking radio or TV but the ,size() indicates groups